### PR TITLE
Update workouts

### PR DESCRIPTION
**Pull request by munichheavies**

### CODE REVIEW

File: data/workouts.json [hunk 1]
```diff
@@ -45,5 +46,31 @@
...
+    "title": "Easy Intervall",
+    "program": [
+      "20 min   @Exa      @20 SPM     2min rest",
+      "6 min    @Exa -4   @22 SPM     2min rest",
+      "20 min   @Exa      @20 SPM     2min rest",
+      "6 min    @Exa -8   @24 SPM"
+    ]
+  },
+  {
+    "title": "Power Hour"
```
Comment (lgeiger) [inline data/workouts.json]: There's a comma missing here

File: data/workouts.json [hunk 2]
```diff
@@ -45,5 +46,31 @@
...
+      "5 min    @6k pace +12   @19 SPM",
+      "15 min   @6k pace +6    @24 SPM",
+      "5 min    @6k pace +12   @19 SPM",
+      "15 min   @6k pace +6    @24 SPM",
+      "5 min    @6k pace +12   @19 SPM",
+      "15 min   @6k pace +6    @24 SPM"
+    ]
+  },
+  {
+    "title": "8x 1min"
```
Comment (lgeiger) [inline data/workouts.json]: There's a comma missing here too

File: data/workouts.json [hunk 3]
```diff
@@ -45,5 +46,31 @@
...
+    "program": [
+      "8x 1min    @max Power    @32+ SPM",
+      "2 min rest"
+    ]
   }
-]
```
Comment (lgeiger) [inline data/workouts.json]: This line should still be there

File: data/workouts.json (hunk 1 above)
Comment (munichheavies) [reply data/workouts.json]: Thanks for fixing mate ;)

Comment (munichheavies) [reply data/workouts.json]: Is it fixed? How do we fix it?

**Pull request closed without merging**